builtin/settingtypes.txt

```diff
@@ -1228,6 +1228,12 @@ language (Language) enum   ,be,cs,da,de,eo,es,et,fr,hu,id,it,ja,jbo,ko,ky,lt,nb,
 #    -    verbose
 debug_log_level (Debug log level) enum action ,warning,action,info,verbose
 
+#    ANSI colored logs: red error log, yellow warning and grey info and verbose logs
+#    "yes" always enables it,
+#    "tty" enables it when printing to terminal,
+#    something else disables it
+log_color (Colored logs) string tty
```

Comment:
Make it an "enum" not a "string", like its described at the start of the file. Then its much nicer in the settings menu.

That would involve giving the "something else" case a special name, like "disabe" or something.